tests/frontend/org/voltcore/zk/StateMachineSnipits.java

```
@@ -879,6 +897,9 @@ public void testStateChangeExample() {
         catch (InterruptedException e) {
             fail();
         }
+        catch (ExecutionException e) {
+            fail();
+        }
```

This will hide the exception if it ever happens you are probably better off letting the exception bubble up and then junit will catch it and fail the test with the exception that caused the error. Same for line 850